cmd/minikube/cmd/env.go

```diff
@@ -144,7 +128,32 @@ type NoProxyGetter interface {
 // EnvNoProxyGetter gets the no_proxy variable, using environment
 type EnvNoProxyGetter struct{}
 
-func generateUsageHint(userShell string) string {
+func generateUsageHint(profile string, userShell string) string {
+	const usgPlz = "Please run command bellow to point your shell to minikube's docker-daemon :"
+	var usgCmd = fmt.Sprintf("minikube -p %s docker-env", profile)
```

Comment:
If they are using the default profile, this should omit the `-p profile`.  Similar to what we do in `start.go`.

Reply:
I see some values that to emphasize specifying profile explicitly. Docker-env command could be confusing enough the more explicit we be the better.

Plus making defualt profile except for the hint text would require a refactoring for the unit tests that would be bigger than this pr.

I am open to do that in a follow up pr